src/java/org/apache/cassandra/security/FileBasedSslContextFactory.java

```diff
@@ -308,12 +308,15 @@ private static String resolvePassword(String keystoreFilePath, String password,
                 return password;
 
             if (StringUtils.isEmpty(passwordFilePath))
-                return null;
+                return password;
```

Comment:
@maulin-vasavada strange change, `password` here will be `null` anyway because if it was not then we would return in the `if` above.

Reply:
Aha I see what you did here. OK. Because `isEmpty(password)` will return false only if it is not null AND String does not have length of 0.

But here you covered the case of returning password of empty string instead of null ...

Reply:
@maulin-vasavada  I have changed my mind: https://issues.apache.org/jira/browse/CASSANDRA-13428?focusedCommentId=17929657&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17929657